components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -389,6 +389,29 @@ bool PublisherInfoDatabase::RestorePublishers() {
   return restore_q.Run();
 }
 
+int PublisherInfoDatabase::GetExcludedPublishersNumber() {
```

Comment:
nit: Can we call `GetExcludedPublishersCount()`?

Reply:
sure

Reply:
done